spinnaker_components/value_source/value_source.c

```diff
@@ -8,12 +8,16 @@ uint n_blocks;            // Number of blocks (in total)
 uint current_block;       // Current block
 value_t* blocks;             // Location of blocks in DRAM
 
+uint us_delay;
+
 void valsource_tick(uint ticks, uint arg1) {
   use(arg1);
   if (simulation_ticks != UINT32_MAX && ticks >= simulation_ticks) {
     spin1_exit(0);
   }
 
+  spin1_delay_us(100);
```

Comment:
Is this so that we're not near the beginning of a time step, so we don't slip from one timestep to another near the beginning of some runs?  If so, a comment to that effect would be handy.